pkg/services/featuremgmt/registry.go

```diff
@@ -397,6 +397,12 @@ var (
 			Description: "Rule backtesting API for alerting",
 			State:       FeatureStateAlpha,
 		},
+		{
+			Name:         "editPanelCSVDragAndDrop",
+			Description:  "Enable drag and drop csv, excel files for edit panel",
```

Comment:
This is not true anymore.

Reply:
Should be fixed now after merging main

Reply:
Updated the registry and generated the files again. Should be good now.